dspace-spring-rest/src/test/java/org/dspace/app/rest/test/AbstractControllerIntegrationTest.java

```diff
@@ -73,6 +73,7 @@
     protected MediaType contentType = new MediaType(MediaTypes.HAL_JSON.getType(),
                                                     MediaTypes.HAL_JSON.getSubtype(), Charsets.UTF_8);
 
+    protected MediaType textUriContentType = new MediaType("text", "uri-list");
```

Comment:
@abollini : we don't need to hardcode "text" and "uri-list" here. We should be able to use the `MediaTypes` constants here as well. e.g.
```
import org.springframework.data.rest.webmvc.RestMediaTypes

protected MediaType textUriContentType = RestMediaTypes.TEXT_URI_LIST;
```

Or, technically, we could drop this `textUriContentType` field altogether.  But, that might require other refactoring.

Reply:
done